tests/wpt/metadata/html/webappapis/scripting/event-loops/microtask_after_script.html.ini

```diff
@@ -1,5 +0,0 @@
-[microtask_after_script.html]
-  type: testharness
-  [Microtask immediately after script]
-    expected: FAIL
-
```

Comment:
Unfortunately this isn't actually a result caused by these changes (see #14221), so we should revert this.